code/modules/mob/death.dm

```diff
@@ -54,6 +54,9 @@
 	if(!gibbed && deathmessage != "no message") // This is gross, but reliable. Only brains use it.
 		src.visible_message("<b>\The [src.name]</b> [deathmessage]")
 
+	for(var/obj/item/organ/O in get_organs())
```

Comment:
Shouldn't this be in ``/mob/living`` or ``/mob/living/carbon`` ? 
Because, organs are mostly only relevant at those levels?

Reply:
Also, unrelated, but I noticed while code diving for this that sillicons seems to bypass calling death in their ``/mob/living/silicon/robot/handle_regular_status_updates()`` line 102.
Dunno if that's a bug, or intended?

Reply:
I mean, they are just getting a hook they can use if they like, makes sense for any type with organs.